gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3-rc-2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4.1-all.zip
```

Comment:
Can you use the normal distribution, please?

Reply:
Ok, will do.

Full distribution was recomended by IntelliJ Idea - https://discuss.gradle.org/t/configuring-gradle-wrapper-to-used-distribution-with-sources-in-intellij-ide/5866